package.json

```diff
@@ -24,13 +26,13 @@
     }
   },
   "dependencies": {
+    "@brafdlog/israeli-bank-scrapers-core": "^0.9.2",
     "@getstation/electron-google-oauth2": "2.1.0",
     "@sentry/electron": "^1.3.0",
     "core-js": "^3.4.4",
     "download-chromium": "^2.2.0",
     "electron-log": "^4.1.1",
     "googleapis": "^49.0.0",
-    "israeli-bank-scrapers-core": "^0.9.1",
```

Comment:
Are you planning to merge your package?

Reply:
Until israeli-bank-scrapers  release the typescript branch. What other option do we have?

Reply:
I don't know.... because I don't know exactly what you did.

I think in TypeScript you can declare untyped modules yourself, like we are doing in the `.d.ts` files.

https://www.typescriptlang.org/docs/handbook/modules.html#working-with-other-javascript-libraries

Reply:
I took the typescript branch in israeli bank scrapers, ran the build and published that as a package. So, when they merge the typescript brach we should just be able to replace my forked package to theirs.
We could start defining modules ourselves but since they already ha a typescript branch that seems to be close to merging i prefer to work over that code instead of doing double work. Makes sense?

Reply:
Yes yes, you are absolutely right, we don't need to do more work here.